inc/BrainPadDisplay.h

```diff
@@ -0,0 +1,27 @@
+#ifndef BPDISPLAY_H
+#define BPDISPLAY_H
+#include "Image.h"
+#include "MbedI2C.h"
+
+namespace codal {
+
+   class BrainPadDisplay {  
+  
+   uint8_t Data[2];
+   int DeviceAddress = 0x78;
+   typedef unsigned char byte;
+
+     public:
+     codal::_mbed::I2C i2c;
```

Comment:
Does this member need to be public?

Reply:
Not sure. We just used the other hardware repos as a template.